package-lock.json

```diff
@@ -30,7 +30,7 @@
         "@material-ui/styles": "^4.11.4",
         "@reach/auto-id": "^0.16.0",
         "@saleor/macaw-ui": "npm:@saleor/macaw-ui@0.7.4",
-        "@saleor/macaw-ui-next": "npm:@saleor/macaw-ui@1.1.15",
+        "@saleor/macaw-ui-next": "npm:@saleor/macaw-ui@0.0.0-pr-20250127234423",
```

Comment:
issue: is this a mistake? if we have unreleased changes we should probably release new macaw version

Reply:
Now it's now, when all test passed I release macaw and bump version to correct one